src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,50 @@
+/* TODO: This is just an example file to illustrate API routing and
+documentation. Can be deleted when the first real route is added. */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
```

Comment:
Nitpick: 
The variable is called controller, and the file it imports from is called controller. 
How much more context is this comment adding? :)